new: Add release notes for Lab v2.11.1 (#433)
memgraph/documentation: pages/release-notes.mdx

</Callout>

## LAb v2.11.1 - Jan 22, 2024

{<h3> New features and improvements </h3>}

> - You can now clearly see the flow of the query plan steps with
>   the arrows on the edges connecting the steps.

Reviewer: ```suggestion
- The flow of the query plan is now easily tracked thanks to the arrows that connect individual steps. 
```

{<h3> Bug fixes </h3>}

> * Importing a file with the new `.cypherl` format that contains a
>   creation of triggers now works as expected.
> * Importing a large `.cypherl` file with more than 1M lines doesn't

Reviewer: ```suggestion
* Importing data from a new CYPHERL file format that contains queries for
  creating triggers now works as expected.
* Importing a large CYPHERL file with more than 1M lines doesn't
```
